typhos/app.py

```diff
@@ -0,0 +1,40 @@
+"""This module defines methods for launching full typhos applications."""
+import logging
+
+from qtpy.QtCore import QTimer
+from qtpy.QtWidgets import QApplication, QMainWindow
+
+from .suite import TyphosSuite
+
+logger = logging.getLogger(__name__)
+qapp = None
+
+
+def get_qapp():
+    """Returns the global QApplication, creating it if necessary."""
+    global qapp
+    if qapp is None:
+        logger.debug("Creating QApplication ...")
+        qapp = QApplication([])
+    return qapp
+
+
+def launch_suite(suite):
+    """Creates a main window and execs the application."""
+    window = QMainWindow()
+    window.setCentralWidget(suite)
+    window.show()
+    logger.info("Launching application ...")
+    QApplication.instance().exec_()
```

Comment:
```suggestion
    get_qapp().exec_()
```

Reply:
The curse of moving old code around is that you draw attention to it!